runtime/command-metrics/pom.xml

```diff
@@ -26,8 +26,8 @@
   <properties>
     <maven.compiler.source>11</maven.compiler.source>
     <maven.compiler.target>11</maven.compiler.target>
-    <jacoco.coverage.ratio>0.35</jacoco.coverage.ratio>
-    <jacoco.missed.count>14</jacoco.missed.count>
+    <jacoco.coverage.ratio>0.50</jacoco.coverage.ratio>
+    <jacoco.missed.count>3</jacoco.missed.count>
```

Comment:
Getting better! 👍

Reply:
Actually after double checking, the missed classes is only 2. But this is as good as its gets, if we want to have higher coverage we have to extract code from `ZillaMetricsCommand` as that has private methods that has some logic inside. Pls LMK I should pursue this.